### PR TITLE
Non-tagger graffiti counts as bad art (and other art adjustments.)

### PR DESCRIPTION
**Pull request by harmonyn**
## About The Pull Request

* Non-taggers' graffiti counts as bad art. Taggers' graffiti counts as good art, not great art.
* Makes the art defines global defines and removes some magic numbers.
* Re-orders the switch statement for art to go ascending instead of descending, because otherwise if something is at BAD_ART it will register as good instead, if it's at GOOD_ART it will register as great, etc., which defeats the purpose of having defines.

## Why It's Good For The Game

The tagger component: Because there isn't really much bad art in the game ,and graffiti is as obvious a candidate as any, and gives the janitor an actual excuse to remove it besides being a dick. If you don't want the potential for a bad moodlet, don't examine the art. You have to go out of your way to do so anyway.

Very slightly nerfing tagger graffiti: Because "what a thought-provoking piece of art" is kind of much for graffiti.

The other component: Because there's no point in having defines if people are just going to put magic numbers everywhere anyway. (I left the impressiveness numbers for statues, etc. alone, because those can take damage, dropping their quality. But if something can't take damage, just use the define.)

## Changelog
:cl:
tweak: Graffiti by a non-tagger counts as bad art.
/:cl:

### CODE REVIEW

**Comment by Fikou** [comment]
#notallgraffiti

**Comment by bgobandit** [comment]
travis what the fuck

**Comment by actioninja** [comment]
lint failure, you're redefining some macros

**Comment by nemvar** [comment]
impressiveness isn't a magic number, its a completely normal one. The defines purely exist to serve as border values for impressiveness.

**Comment by bgobandit** [comment]
yeah but the graffiti wasn't using it before

**Comment by Michael-Ashfield** [comment]
So random jokers spraying fake runes in maint will now actively give sec and the detective bad moodlets when examined?

**Comment by bgobandit** [comment]
very minor and short-lived bad moodlets, but yes

**Comment by Onule** [comment]
> Very slightly nerfing tagger graffiti: Because "what a thought-provoking piece of art" is kind of much for graffiti.

so my monster girls aren't thought-provoking now? doing me a sad here.

**Comment by optimumtact** [comment]
```
code/game/objects/structures/statues.dm, line 2, column 9:

warning: macro redefined: BAD_ART

- code/datums/components/art.dm:1:9: previous definition of BAD_ART

code/game/objects/structures/statues.dm, line 3, column 9:

warning: macro redefined: GOOD_ART

- code/datums/components/art.dm:2:9: previous definition of GOOD_ART

code/game/objects/structures/statues.dm, line 4, column 9:

warning: macro redefined: GREAT_ART

- code/datums/components/art.dm:3:9: previous definition of GREAT_ART

```

File: code/datums/components/art.dm [hunk 1]
```diff
@@ -40,8 +41,4 @@
 	to_chat(M, "You start examining [parent].")
 	if(!do_after(M, 20, target = parent))
 		return
-	on_obj_examine(source, M)
-
-#undef BAD_ART
-#undef GOOD_ART
-#undef GREAT_ART
+	on_obj_examine(source, M)
```
Comment (optimumtact) [inline code/datums/components/art.dm]: you need those undefs probably

Comment (bgobandit) [reply code/datums/components/art.dm]: I got rid of the undefs so that the defines can actually be used for reasons above

Comment (optimumtact) [reply code/datums/components/art.dm]: yeah well you're breaking the linter by reusing macros

**Comment by optimumtact** [comment]
okay, now put the defines in a file in __DEFINES :grin:

**Pull request merged**